### ISSUE TITLE
Doesn't work (dev version of ggplot?)

### ISSUE THREAD

**Issue by carbonmetrics** [drsimonj/corrr#24]
Doesn't work for me. I have installed the latest development version of ggplot2 and all my ggplot extensions stopped to work.  The developer of ggplot2 thinks this is [tough luck](https://github.com/hadley/ggplot2/issues/1809) for the developers of those extensions. 

```
airquality %>% correlate() %>% network_plot()
Warning: Ignoring unknown aesthetics: x, y
Error in zero_range(from) : x must be length 1 or 2

sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.1 LTS

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=nl_NL.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=nl_NL.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=nl_NL.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=nl_NL.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] corrr_0.2.1 dplyr_0.5.0

loaded via a namespace (and not attached):
 [1] ggrepel_0.5        Rcpp_0.12.7        assertthat_0.1     grid_3.3.1         R6_2.2.0           plyr_1.8.4         gtable_0.2.0       DBI_0.5-1          magrittr_1.5       scales_0.4.0       ggplot2_2.1.0.9001 lazyeval_0.2.0    
[13] labeling_0.3       tools_3.3.1        munsell_0.4.3      colorspace_1.2-6   tibble_1.2        
```

BTW - have you seen the `DescTools::PlotWeb` function?

**Comment by drsimonj** [comment]
Thanks @carbonmetrics. I'm aware of this (Hadley Wickham kindly alerted all authors of packages that depend on ggplot2 with any errors). The error you're experiencing is due to a conflict between the dev ggplot2 (2.1.0.9001) and the stable CRAN release of ggrepel (0.5). However, the latest version of ggrepel on GitHub works perfectly. My goal is to add a conditional into the script to handle this. In the meantime, you have two options:

1. Role back to stable ggplot2 version 2.1.0
2. Install latest tagged version of ggrepel from GitHub via `devtools::install_github("slowkow/ggrepel@0.6.2")`

**Comment by drsimonj** [comment]
Closed with f970550
